parsec/backend/pki.py

```diff
@@ -177,7 +192,7 @@ async def api_pki_enrollment_info(
         try:
             info = await self.info(
                 organization_id=client_ctx.organization_id,
-                enrollment_id=msg["enrollment_id"],
+                enrollment_id=EnrollmentID(cast(UUID, msg["enrollment_id"])),
```

Comment:
Will be outdated with the PR on the rework of `uuid` on python

Reply:
Should we wait #3587 to be merged? What do you think @FirelightFlagboy @TimeEngineer